ccdImports/configFiles/CCD_Probate_Legacy_Search/CaseEvent.json

```diff
@@ -1,4 +1,5 @@
 [
   {"LiveFrom": "01/01/2017", "CaseTypeID": "LegacySearch", "ID": "createDummy", "Name": "Create a dummy search case", "Description": "Create a dummy search case", "DisplayOrder": 1, "PostConditionState": "dummyCaseCreated", "SecurityClassification": "Public", "ShowSummary": "Y", "EndButtonLabel": "Save and continue"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "LegacySearch", "ID": "legacySearch", "Name": "Search for legacy cases", "Description": "Search for legacy cases", "DisplayOrder": 2, "PreConditionState(s)": "*", "PostConditionState": "*", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/legacy/resetSearch", "SecurityClassification": "Public", "ShowSummary": "Y"}
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "LegacySearch", "ID": "legacySearch", "Name": "Search for legacy cases", "Description": "Search for legacy cases", "DisplayOrder": 2, "PreConditionState(s)": "*", "PostConditionState": "*", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/legacy/resetSearch", "SecurityClassification": "Public", "ShowSummary": "Y"},
+  {"LiveFrom": "16/03/2023", "CaseTypeID": "LegacySearch", "ID": "resendData", "Name": "Resend Data", "Description": "Resend Iron Mountain Data feed", "DisplayOrder": 3, "PreConditionState(s)": "*", "PostConditionState": "*", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/data-extract/resend-iron-mountain", "SecurityClassification": "Public", "ShowSummary": "Y"}
```

Comment:
can we rename to dataExtractResendDate - I assume we will use the same field for any type of future data extract resend

Reply:
changing from resendData to dataExtractResendData (assuming that was a typo)